system/i18n/cms.properties

```diff
@@ -560,6 +560,7 @@ ckeditor.imagepicker.spacing_right.label=Right (px)
 ckeditor.imagepicker.spacing_bottom.label=Bottom (px)
 ckeditor.imagepicker.spacing_left.label=Left (px)
 
+ckeditor.imagepicker.image_derivative.label=Image Derivative
```

Comment:
I think we should rename this to "Preset". "Image Derivative" is perhaps too technical for end-users.